<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyAudit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>TeX Gyre Adventor</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Remediation Scripts</string>
    </property>
   </widget>
   <widget class="QListWidget" name="remediationlistWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>731</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runRemediation">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Scripts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skipRemediation">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
